m2c/types.py

```diff
@@ -927,6 +927,16 @@ def u64() -> Type:
     def int64() -> Type:
         return Type(TypeData(kind=TypeData.K_INT, size_bits=64))
 
+    @staticmethod
+    def s128() -> Type:
+        return Type(TypeData(kind=TypeData.K_INT, size_bits=128, sign=TypeData.SIGNED))
+
+    @staticmethod
+    def u128() -> Type:
+        return Type(
+            TypeData(kind=TypeData.K_INT, size_bits=128, sign=TypeData.UNSIGNED)
+        )
+
```

Comment:
no need for these (128-bit types aren't used for arithmetic so they don't make much sense)